integration/nydus/nydus_tests.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Ant Group
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This will test the nydus feature is working properly
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+dir_path=$(dirname "$0")
+source "${dir_path}/../../lib/common.bash"
+source "${dir_path}/../../.ci/lib.sh"
+#cidir="${dir_path}/../../.ci"
+#echo "cidir $cidir"
+KATA_HYPERVISOR="${KATA_HYPERVISOR:-qemu}"
+
+if [ "$KATA_HYPERVISOR" != "qemu" ]; then
+	echo "Skip nydus test for $KATA_HYPERVISOR, it only works for QEMU now."
```

Comment:
Might be worth mentioned the GH issue for this: we probably need an issue / hypervisor.

Reply:
added an issue link in the output message.